packages/nx/src/command-line/graph/graph.ts

```diff
@@ -256,7 +256,7 @@ export async function generateGraph(
     if (ext === '.html') {
       const assetsFolder = join(fileFolderPath, 'static');
       const assets: string[] = [];
-      copySync(join(__dirname, '../../core/graph'), assetsFolder, {
+      copySync(join(__dirname, '../core/graph'), assetsFolder, {
```

Comment:
These paths seem incorrect now. This would resolve to packages/nx/src/command-line/core, instead of packages/nx/src/core right?